kvs/src/kvs/user_request_handler.cpp

```diff
@@ -78,6 +78,11 @@ void user_request_handler(
         KeyTuple* tp = response.add_tuples();
         tp->set_key(key);
 
+        if (!is_metadata(key)) {
```

Comment:
Do we want to always log the keys being processed? Or is this for debugging only?

Reply:
I intended for this to be general logging, but I don't feel strongly about it if you think it should be removed.